Makefile

```diff
@@ -8,19 +8,13 @@ linter:=$(shell which golangci-lint 2>/dev/null || echo $(HOME)/go/bin/golangci-
 build: fioctl-linux-amd64 fioctl-linux-arm64 fioctl-windows-amd64 fioctl-darwin-amd64 fioctl-darwin-arm64
 	@true
 
-fioctl-static:
-	CGO_ENABLED=0 go build -a -ldflags '-w -extldflags "-static"' -o ./bin/fioctl-static ./main.go
-
 # Allows building a dyn-linked fioctl on platforms without pkcs11-tool (not built by default)
 fioctl-cgo-pkcs11:
 	CGO_ENABLED=1 go build -tags cgopki $(LDFLAGS) -o bin/$@ ./main.go
 
-fioctl-linux-amd64:
-fioctl-linux-arm64:
-fioctl-linux-armv7:
-fioctl-windows-amd64:
-fioctl-darwin-amd64:
-fioctl-darwin-arm64:
+# Any target of "fioctl-{GOOS}-{GOARCH}" can be built for your desired platform.
+# See "build" for a list of platforms supported and verified by Foundries.io.
+# Set GOARM={version} when compiling for specific Arm version; default is to auto-detect or 6 if cannot detect.
```

Comment:
Taken from here: https://go.dev/doc/install/source#environment
Maybe it is worth to reference it instead of copy-pasting a part of that content.

Reply:
what's nice about having them is that you get auto-completion when running `make <tab>`

Reply:
hmm... I never thought about that. Always doing `^R` instead.

Reply:
I can revert this one and only remove the armv7 (which does not work anyway).

Reply:
reverted in force-push